geom/src/polyline.rs

```diff
@@ -89,6 +89,17 @@ impl PolyLine {
         self.check_empty()
     }
 
+    /**
+    Project p onto the polyline and return the distance from the projection to p
+    **/
+    pub fn project_len(&self, p: Vec2) -> f32 {
+        let proj = self.project(p);
+        proj.distance2(p)
```

Comment:
it's proj.distance not proj.distance2 which is the distance squared, I should rename it to distance_sq and same goes for magnitude2

Reply:
I intended it to be project2_len, so I'm going to change the method name. I think the "2" style notation is fine -- it's pretty commonly used that way in other geom libraries.